modules/core/05-port/client/cli/cli.go

```diff
@@ -0,0 +1,25 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/ibc-go/modules/core/05-port/types"
+)
+
+// GetQueryCmd returns the query commands for IBC channels
```

Comment:
godoc needs updating. Could you fix the imports as well?

Reply:
Sure. Do you want to leave this `cli.go` in place for #112 and remove the cmd for version negotiation? 
Alternatively I can remove the entire `client/cli` and re-add later

Reply:
lets remove entire cli and re-add later